### PR TITLE
Milestone 9: Pop-Up Message

### PR DESCRIPTION
**Pull request by eroiyo**
Added Pop-Up Messages! they appear when you click on "See Project" buttons of the project cards...

-They have extended information
-Blur the background back them
-Feature a different size image

### CODE REVIEW

File: js/data.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    overview: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting  ever since the 1500s, when an unknown printer took a galley of type veris lapoa todoe.",
+    tags: 'Css,Html,Bootstrap,Ruby',
+    live: '#',
+    repo: '#',
+    alt: 'placeholder',
+    bAlt: 'Big_Placeholder',
+  },
+];
+export default data;
```
Comment (vichuge) [inline js/data.js:55]: - [ ] It looks like all the data are exactly the same on the script, to use different information for each project make it easier to see bugs and have a project more like it should be work, I invite you to change the info and fix the necessary issues on your project after that 🐶

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

mario_ladders

## Good points 👍 
- No Linters error ✔️
- Your design is good ✔️ 

## Aspects to improve ♻️

- [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by eroiyo** [comment]
Added Different elements, so now each element have their own name, overview and description 📖

File: js/data.js (hunk 1 above)
Comment (eroiyo) [reply js/data.js]: Thanks for the Review!

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @eroiyo, 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**